promscale/promscale-benefits.md

```diff
@@ -0,0 +1,164 @@
+# Promscale benefits
+
+**Get started in minutes**: deploy a full observability stack for metrics and
+traces in your Kubernetes cluster with a single command using [our cli](tobs). Or deploy
```

Comment:
broken link

Reply:
i'll have to fix this when we move the tobs content over.

Reply:
We already have the tobs page in the install section maybe we start referring to it. :)